### PR TITLE
{bio}[foss/2022b] CheckM2 v1.0.2

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
TODO: these 2 commands needs to be add to easyconfig?
- `checkm2 database --download --path $EBROOTCHECKM2`
- `export CHECKM2DB="$EBROOTCHECKM2/CheckM2_database/uniref100.KO.1.dmnd"`

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/c/CheckM2/CheckM2-1.0.2-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    }),
+]
+
+postinstallcmds = [
+    # np.float is depreciated in newer numpy
+    'sed -i "s/np.float/float/g" %(installdir)s/lib/python%(pyshortver)s/site-packages/checkm2/predictQuality.py',
+    # update DB_LOCATION_DEFINITION in defaultValues.py to env CHECKM2DB
+    'sed -i "31d" %(installdir)s/lib/python%(pyshortver)s/site-packages/checkm2/defaultValues.py',
+    "sed -i '30 a\    DB_LOCATION_DEFINITION = os.environ.get(\"CHECKM2DB\", \"Not Set\")' " +
+    "%(installdir)s/lib/python%(pyshortver)s/site-packages/checkm2/defaultValues.py",
```
Comment (boegel) [inline easybuild/easyconfigs/c/CheckM2/CheckM2-1.0.2-foss-2022b.eb:48]: make this a raw string: `r"..."`

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/CheckM2/CheckM2-1.0.2-foss-2022b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/c/CheckM2/CheckM2-1.0.2-foss-2022b.eb:48]: Maybe better (so you can avoid `unix2dos`):

```
"cd %(installdir)s/lib/python%(pyshortver)s/site-packages/checkm2 && "
r"sed -i 's/\(DB_LOCATION_DEFINITION\) = .*/\1 = os.environ.get(\"CHECKM2DB\", \"Not Set\")' defaultValues.py"

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3155.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/04cd83854307fb03e1be965f277cd8d2 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**